airflow/timetables/base.py

```diff
@@ -79,6 +74,18 @@ def interval(cls, start: DateTime, end: DateTime) -> "DagRunInfo":
         """
         return cls(run_after=end, data_interval=DataInterval(start, end))
 
+    @property
+    def schedule_date(self) -> DateTime:
```

Comment:
No, this is the “logical” date of the DagRun, not the earliest time to schedule it.

The equivalence of `run_date` in the proposal is `run_after` in the implementation (because I simply forgot `run_date` was in the proposal…); we can change that if needed.